packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2650,6 +2650,11 @@ export class StripeHelper extends StripeHelperBase {
     };
   }
 
+  stripePlanToPaymentCycle(plan: Stripe.Plan) {
+    if (plan.interval_count === 1) return plan.interval;
+    return `${plan.interval_count} ${plan.interval}`;
```

Comment:
Could we pluralize the `plan.interval` here? The email text still reads a little awkward, but a little less so.